mount/default.conf

```diff
@@ -21,16 +21,34 @@
 # Use cvmfs_config showconfig to get the effective parameters.
 #
 
+# The client profile selects configuration options tuned for certain use cases
+# Overwrite this parameter in /etc/cvmfs/default.local
+# Set CVMFS_CLIENT_PROFILE=single if you run cvmfs on a laptop or a small
```

Comment:
I'm not sure it is appropriate to refer to a variable in the cvmfs-config-default package in /etc/cvmfs/default.conf.

Reply:
I'd like to establish the `CVMFS_CLIENT_PROFILE` variable for the benefit of all the config repositories.  I think it would be generally useful to have a hint how the client is intended to be used.

Reply:
If you'd like to establish it as a default for other config repos, ok, we can update those, but then please remove "from the cvmfs-config-default package" from the comment.  You could say instead "if supported by the cvmfs-config package".

Should CVMFS_USE_CDN be similarly documented then, for consistency?

Note for the record that I think that CVMFS_USE_CDN=yes should be the default (at least in config repo) when CVMFS_CLIENT_PROFILE=single.  That's effectively what the OSG & EGI config repos already do (implied when CVMFS_HTTP_PROXY isn't set) and what I intend for them to continue to do.  It protects the stratum 1s and gives a better user experience.  When we discussed it you disagreed.